.github/pull_request_template.md

```diff
@@ -6,10 +6,8 @@
 <!--
 If the validation checks fail
   1. Run `make check-types` (from the root directory of the repository) and fix any mypy (https://mypy.readthedocs.io/en/stable/) errors.
-
-  2. Run `make check-style` and fix any flake8 (http://flake8.pycqa.org) errors.
-
-  3. Run `make format` to format your code with the black (https://black.readthedocs.io/en/stable/index.html) autoformatter.
+  2. Run `make check-format` and fix any ruff linting errors.
+  3. Run `make format` to format your code.
```

Comment:
Do we want to run `make format` AFTER `make check-format`? I would imagine we would want to call `make format` to automatically correct things before any manual intervention is required.

Reply:
That's a good point. I've added 787cab2a to simplify this.